app/models/workers_scaling/experiment_statistics.rb

```diff
@@ -85,8 +91,10 @@ def makespan(params = {})
     #   * opts
     #   * cond
     # Possible cond and opts can be found in MongoActiveRecord#where.
+    # By default only running workers are included into calculations.
     # Raises InfrastructureError
     def get_infrastructure_statistics(infrastructure, params = {})
+      params[:cond] = Query::RUNNING_WORKERS unless params.has_key? :cond-
```

Comment:
Trochę mi się nie podoba, że to jest dopisane do każdej metody, ale nie mam pomysłu jak poprawić. Ten minus na końcu nie wyrzuca przypadkiem błędu, że nie można odjąć hasha od symbolu?